server/tso/allocator_manager.go

```diff
@@ -291,12 +302,12 @@ func (am *AllocatorManager) getAllocatorPath(dcLocation string) string {
 
 // similar logic with leaderLoop in server/server.go
 func (am *AllocatorManager) allocatorLeaderLoop(ctx context.Context, allocator *LocalTSOAllocator) {
+	defer log.Info("server is closed, return local tso allocator leader loop",
```

Comment:
There are some other cases that will return

Reply:
According to the codes, the only case that will make the `allocatorLeaderLoop` return is that `ctx` is canceled, which can only be caused by the server closing. Which other cases do you mean exactly?

Reply:
Oh, you're right, my mistake